src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { ClusterService } from '~/app/shared/api/cluster.service';
+import { OrchestratorService } from '~/app/shared/api/orchestrator.service';
+import { AppConstants } from '~/app/shared/constants/app.constants';
+import { NotificationType } from '~/app/shared/enum/notification-type.enum';
+import { Permission } from '~/app/shared/models/permissions';
+import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
+import { NotificationService } from '~/app/shared/services/notification.service';
+
+@Component({
+  selector: 'cd-create-cluster',
+  templateUrl: './create-cluster.component.html',
+  styleUrls: ['./create-cluster.component.scss']
+})
+export class CreateClusterComponent implements OnInit {
+  permission: Permission;
+  orchStatus = false;
+  featureAvailable = false;
+  projectConstants: typeof AppConstants;
```

Comment:
As this doesn't go through dep-injection, why not assigning it statically?
```suggestion
  projectConstants: typeof AppConstants = AppConstants;
```

Reply:
Done.